### PR TITLE
docs: addLabels works for spans as well

### PR DESCRIPTION
**Pull request by jahtalab**

### CODE REVIEW

File: docs/agent-api.asciidoc [hunk 1]
```diff
@@ -86,7 +86,8 @@ deprecated[4.1.x,Tags have been replaced with labels], Please use <<apm-add-labe
 apm.addLabels({ [name]: value })
 ----
 
-Set labels on transactions and errors.
+Set labels on transactions and errors. 
+Starting with APM Server 7.6+, the labels are added to spans as well.
```
Comment (vigneshshanmugam) [inline docs/agent-api.asciidoc]: this happen on the server right? For global labels?

Comment (jahtalab) [reply docs/agent-api.asciidoc]: That's correct!

Comment (vigneshshanmugam) [reply docs/agent-api.asciidoc]: Do you want to link to the server docs? just an idea

Comment (jahtalab) [reply docs/agent-api.asciidoc]: I don't think that's necessary since the user shouldn't need to care about how it works behind the scene.

**Comment by jahtalab** [comment]
Merging this as failures are unrelated to the changes

**Pull request merged**